account/src/main/java/com/pyokemon/account/auth/service/AccountService.java

```diff
@@ -67,7 +67,7 @@ public LoginResponseDto login(LoginRequestDto request, String role) {
         String refreshToken =
                 tokenGenerator.generateRefreshToken(account.getAccountId(), account.getRole());
 
-        if (role.equals("USER")) {
+        if (account.getRole().equals("USER")) {
```

Comment:
```suggestion
        if ("USER".equals(account.getRole())) {
```
사소하지만 NPE 방지 위해 습관 들이는 것 좋아요~!